paimon-hive/paimon-hive-catalog/src/main/java/org/apache/paimon/hive/HiveCatalog.java

```diff
@@ -479,7 +479,8 @@ private Table newHmsTable(Identifier identifier) {
         return table;
     }
 
-    private void updateHmsTable(Table table, Identifier identifier, TableSchema schema) {
+    private void updateHmsTable(Table table, Identifier identifier, TableSchema schema)
```

Comment:
`UnsupportedOperationException` is a runtime exception, there is no need to throw here.

the same below.

Reply:
thanks ,I update it